tests/Fixer/Phpdoc/PhpdocTagTypeFixerTest.php

```diff
@@ -348,6 +348,36 @@ public function provideFixCases()
                 '<?php
 /**
  * @return array{0: float, 1: int}
+ */',
```

Comment:
Could you add a case like the failing one you reported in #5815, i.e. `@internal ending with a {@see tag}`?